app/common/components/table.njk

```diff
@@ -11,10 +13,16 @@
     {% for row in params.data %}
       <div data-test-id="table-row-{{ loop.index0 }}" class="nhsuk-grid-row nhsuk-u-margin-1 nhsuk-u-padding-top-3 nhsuk-u-padding-bottom-3 bc-t-border-bottom--grey-3">
         {% for dataPoint in row %}
-          {% if params.columnInfo[loop.index0].link %}
-            <a class="nhsuk-u-padding-1 {{ params.columnClass }}" href="#">{{ dataPoint }}</a>
+          {% if dataPoint.href %}
+            <a class="nhsuk-u-padding-1 {{ params.columnClass }}" href="{{ dataPoint.href }}">{{ dataPoint.data }}</a>
+          {% elif dataPoint.tag %}
+            {{ tag({
+              dataTestId: dataPoint.tag.dataTestId,
+              text: dataPoint.tag.text,
+              classes: dataPoint.tag.classes
+            }) }}
           {% else %}
-            <div class="nhsuk-u-padding-1 {{ params.columnClass }}">{{ dataPoint }}</div>
+            <div class="nhsuk-u-padding-1 {{ params.columnClass }}">{{ dataPoint.data }}</div>
```

Comment:
can use our view-data-text component here?

Reply:
I haven't changed the implementation btw - I did this component last PR and it is merged ...

Reply:
I haven't changed the implementation btw - I did this component last PR and it is merged ...